README.md

```diff
@@ -12,16 +12,16 @@ using the preview.
 
```

Comment:
Guess prettier just formatted  in vscode, let me know if we don't want to do that.